ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/AccessListEntry.java

```diff
@@ -70,4 +79,29 @@ public void serialize(
       gen.writeEndObject();
     }
   }
+
+  public static class Deserializer extends StdDeserializer<AccessListEntry> {
```

Comment:
Style for later: JSON representation is a different design concern from the code use of access lists and should be handled in the JSON-RPC packages.  In theory if we dropped JSON-RPC we would then have to touch this class, which indicates the design covers too much space.

I see there is already a serailzier.

Since this is shared between JSON and the EVMTool test representation a stand alone package would be appropriate.